docs/sync.js

```diff
@@ -148,9 +149,10 @@ class User {
      * @param {string} options.provider - The provider type
      * @param {string} options.providerToken - The access token for the given provider
      * @param {object} [options.userInfo] - A map containing additional data required by the provider
-     * @param {function(error, User)} callback - called with the following arguments:
+     * @param {function(error, User)} [callback] - an optional called with the following arguments:
```

Comment:
"optional" -> "optional function"???

Reply:
that got blindly copied from the other PR, can reword it